pdns/syncres.cc

```diff
@@ -2366,6 +2397,13 @@ RCode::rcodes_ SyncRes::updateCacheFromRecords(unsigned int depth, LWResult& lwr
       }
     }
 
+    if (recordState == Bogus) {
+      /* this is a TTD by now, be careful */
+      for(auto& record : i->second.records) {
+        record.d_ttl = std::min(record.d_ttl, static_cast<uint32_t>(s_maxbogusttl + d_now.tv_sec));
```

Comment:
isn't `record.d_ttl` TTL and not a TTD ?

Reply:
Unless I'm mistaken it's a TTD at this point, since `dr.d_ttl += d_now.tv_sec;` line 2283.